src/components/cryptoItem/cryptoStyle.ts

```diff
@@ -0,0 +1,85 @@
+//@ts-ignore
+import styled from 'styled-components/native';
+import {colors} from '../utils/colors';
+
+export const Contener = styled.View`
+  padding-top: 7px;
+  padding-bottom: 8px;
+  display: flex;
+`;
+
+export const Separador = styled.View`
+  height: 1px;
+  width: 100%;
+  margin-top: 11px;
+  background-color: ${colors.dividergray};
+`;
+
+export const OneCrypto = styled.View`
+  width: 88%;
+  height: 100px;
+  margin-bottom: 10px;
+  border-bottom: 0.2px;
+  border-bottom-color: ${colors.dividergray};
+  display: flex;
+  align-self: center;
+`;
+
+export const NamesContainer = styled.View`
+  width: 25%;
+  position: relative;
+  left: 18%;
+  bottom: 14%;
+`;
+
+export const Name = styled.Text`
+  color: ${colors.black};
+  font-size: 17px;
+  font-weight: 500;
+`;
+
+export const Symbol = styled.Text`
+  color: ${colors.gray};
+  font-size: 15px;
+`;
+
+export const Logo = styled.Image`
+  width: 48px;
+  height: 48px;
+  position: relative;
+  left: 5px;
+  top: 30px;
+`;
+
+export const ValueContener = styled.View`
+  display: flex;
+  flex-direction: column;
+  text-align: right;
+  margin-top: -55px;
```

Comment:
did you make this negative margin to align the texts? if so, check the styles because it is not necessary, remember to use flex.

Reply:
this.